src/tests/issues.targets

```diff
@@ -1166,6 +1166,16 @@
         </ExcludeList>
     </ItemGroup>
 
+    <!-- NativeAOT arm32 specific excludes -->
+    <ItemGroup Condition="'$(XunitTestBinBase)' != '' and '$(TestBuildMode)' == 'nativeaot' and '$(RuntimeFlavor)' == 'coreclr' and ('$(TargetArchitecture)' == 'arm' or '$(AltJitArch)' == 'arm')">
+        <ExcludeList Include="$(XunitTestBinBase)/Loader/classloader/Casting/punning*">
+            <Issue>https://github.com/dotnet/runtime/issues/98493</Issue>
+        </ExcludeList>
+        <ExcludeList Include="$(XunitTestBinBase)/JIT/Directed/nullabletypes/Desktop/boxunboxvaluetype_*">
+            <Issue>https://github.com/dotnet/runtime/issues/95517</Issue>
+        </ExcludeList>
```

Comment:
Seems like I made some mistake and the exclusions don't work as intended. :-/

I submitted a fix for the second one (#98547).

Reply:
Let's remove the exclusions and do another test run :-)

Reply:
> I submitted a fix for the second one (https://github.com/dotnet/runtime/pull/98547).

#98547 fix is not correct as discovered in https://github.com/dotnet/runtime/pull/98623#discussion_r1493482499 . I expect that the test will keep failing.

Also, there may be unforeseen perf issues due to https://github.com/dotnet/runtime/pull/98623#issuecomment-1951088692 . I am thinking that we should revert it and wait for the proper fix from @EgorBo

Reply:
> I expect that the test will keep failing.

The test doesn’t fail. I checked that locally before submitting the fix. However, it doesn’t fail because the size of the block is small enough to take the managed code path which happens to trigger NRE correctly. It would still fail for big sizes but that’s not the case in the tested unboxing scenario.